external/systemjs/plugin-babel-cached.js

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var babel = require('plugin-babel');
+
+var cacheExpiration = 30 /* min */ * 60 * 1000;
```

Comment:
Given that the runtime on the Linux bot is already >30 minutes, and that PR #8284 will make the tests slightly slower, would it perhaps make sense this use `60 minutes` instead to avoid cache invalidation during test runs?